machines-endpoints/pkg/machines-endpoints/main.go

```diff
@@ -418,4 +488,16 @@ func main() {
 			log.ErrorExit(err)
 		}
 	}
+
+	if *flgBMCLogCollectorConfigMap {
+		// create bmc-log-collector configmap of all servers
+		addersses, err := createMachinesList(machines)
+		if err != nil {
+			log.ErrorExit(err)
+		}
+		err = client.updateBmcLogCollectorConfigMap(ctx, addersses)
+		if err != nil {
+			log.ErrorExit(err)
+		}
```

Comment:
Variable name `addresses` is not appropriate.

```golang
		// create bmc-log-collector configmap of all servers
		machinesList, err := createMachinesList(machines)
		if err != nil {
			log.ErrorExit(err)
		}
		err = client.updateBmcLogCollectorConfigMap(ctx, machinesList)
		if err != nil {
			log.ErrorExit(err)
		}
```
or
```golang
		// create bmc-log-collector configmap of all servers
		str, err := createMachinesList(machines)
		if err != nil {
			log.ErrorExit(err)
		}
		err = client.updateBmcLogCollectorConfigMap(ctx, str)
		if err != nil {
			log.ErrorExit(err)
		}
```

Reply:
Changed